b3/parser.py

```diff
@@ -326,7 +326,7 @@ def __init__(self, conf, options):
             game_log = self.config.get('server', 'game_log')
             if game_log[0:6] == 'ftp://' or game_log[0:7] == 'sftp://' or game_log[0:7] == 'http://':
                 self.remoteLog = True
-                self.bot('Working in remote-log mode: %s', game_log)
+                self.bot('Working in remote-log mode')
```

Comment:
What is the purpose of this change? Why hiding the gamelog path from the B3 log?

Reply:
Because it shows the SFTP/FTP password within the log path.

Reply:
Fair enough 👍